pkgs/development/libraries/libdatrie/default.nix

```diff
@@ -42,5 +42,6 @@ stdenv.mkDerivation rec {
     license = licenses.lgpl21Plus;
     platforms = platforms.unix;
     maintainers = with maintainers; [ ];
+    pkgConfigModules = [ "datrie-2.0" ];
```

Comment:
```suggestion
    pkgConfigModules = [ "datrie-0.2" ];
```

Reply:
Slightly flabbergasted how this annotation fixed my build although it was wrong.

Reply:
Its actually not the first time I noticed that. For libxcb I accidentally put "xcb.pc" in the pkgConfigModules list and it still fixed my dependency.

Reply:
Oh, we are only checking whether the attribute exists …